src/index.html

```diff
@@ -11,6 +11,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="link"
+      method="get"
+      class="formBig"
```

Comment:
No needs to have to different classes as formBig and formSmall
You can use BEM methodology
Take one basic class - form
And for another form just add modifier like --small

Apply all over the document